api/genomics/src/main/java/org/pmiops/workbench/genomics/RandomizeVcf.java

```diff
@@ -50,9 +51,9 @@ public RandomizeVcf() {
   }
 
   @VisibleForTesting
-  protected RandomizeVcf(String sampleNameSuffix, Random random) {
+  protected RandomizeVcf(int samples, Random random) {
     super();
-    this.sampleNameSuffix = sampleNameSuffix;
```

Comment:
nit: maybe add a prefix to indicate that its an int? I thought it was a list while reading through

Reply:
How about `numberOfSamples`?